contracts/CertiKSecurityOracle.sol

```diff
@@ -42,19 +42,38 @@ contract CertiKSecurityOracle is Ownable {
     }
   }
 
+  function getSecurityScoreBytes4(address contractAddress, bytes4 functionSignature)
```

Comment:
this is a breaking change. but since the contract hasn't been released yet, and in README we didn't expose this one, it should be fine.